doc/deployment_string.md

```diff
@@ -62,5 +66,5 @@ A deployment string JSON would look like this:
 When stringified, the above example would look like:
 
 ```
-"deployment": "{\"services\":{\"gps\":{\"image\":\"openhorizon/x86/gps:2.0.3\",\"privileged\":true,\"environment\":[\"FOO=bar\"],\"devices\":[\"/dev/bus/usb/001/001:/dev/bus/usb/001/001\"],\"binds\":[\"/tmp/testdata:/tmp/mydata:ro\",\"myvolume1:/tmp/mydata2\"],\"ports\":[{\"HostPort\":\"5200:6414/tcp\",\"HostIP\":\"0.0.0.0\"}]}}}"
+"deployment": {"services":{"gps":{"image":"openhorizon/x86/gps:2.0.3","privileged":true,"environment":["FOO=bar"],"devices":["/dev/bus/usb/001/001:/dev/bus/usb/001/001"],"binds":["/tmp/testdata:/tmp/mydata:ro","myvolume1:/tmp/mydata2"],"ports":[{"HostPort":"5200:6414/tcp","HostIP":"0.0.0.0"}],"tmpfs":{"/app":""}}}}
```

Comment:
Please do not remove the outer quotes and internal backslashes because the deployment in the exchange is a string. So it has to be quoted.